research-activities/card-sort.md

```diff
@@ -1,20 +1,42 @@
-# Card sorts
+# Card sorting
 
-**Things you'll need:** Index cards, post-it notes, pens, a large table, 4-5 participants
+**Things you'll need:** Index cards, post-it notes, pens, a large table, 3-5 participants, an observer/note taker, a smartphone for pictures. 
 
 **Estimated time:** 60 minutes
 
 ## Best to use when
+Card sorts are a good way to figure out what to call things and how they should be organised so that users can find what they are looking for. It can also be used to prioritise what to work on in a given sprint and can help teams identify missing user needs. 
 
-Card sorts are a good way to figure out how information or transactions should be organised so that most users will understand where to find things.
+**There are two types of card sorts:**
 
-## Instructions
+Open - participants group sample content, ideas, needs and come up with a name for the category. 
 
-1. On the index cards, participants write labels representing subject areas of user need, interest or existing pages.
-2. Participants then group the cards and give each group a name. You can prepare some group names before the day, or just let participants make them up.
-3. Talk to participants to understand why they have grouped things they way they have
-4. Ask participants to rank the groups in order of priority
+Closed - participants move content, ideas, and needs to already named categories. This is also useful for later stages of  designing something, when content is revised and a well-tested structure is already in place. You can also use closed card sorting to prioritise needs, features and products.
 
-## Examples
+You can also think of these as two parts of the design process:
+Use open sorts to understand what goes together and what it should be called. 
+Use closed sorts to test and validate what you already have with more people and other types of users. 
 
-TODO: we need some photographs and example outputs.
+Hybrid sessions are also useful. You provide categories but let people create new ones, and remove and rename categories if they feel it doesn’t fit into any of the existing ones.
+
+## Instructions (open sort)
+
+1. Prepare category cards for each participant. *Write categories on cards for a closed sort.
+2. Prepare some content cards for each category (a set for each participant)
+3. Ask participants to group the cards in the way that makes sense to them and give the groups names. *For a closed sort participants move content cards into categories you’ve prepared.
+4. Discuss why they grouped the cards the way they did
```

Comment:
Presumably the note taker whirls into action at this point. Any tips for them at this point, what are they trying to record? Perhaps a link to your note taking piece would set this up nicely?